src/components/SingleSourceRow.vue

```diff
@@ -22,16 +22,18 @@
 
 <template>
 
-  <div class="swipe-wrapper" v-if="rowObject">
-    <article class="card-tile" v-for="(value, key) in rowObject" :key="key">
-      <a class="article-anchor-wrapper" :href="value?.article_url">
-        <span class="article-image-wrapper" :style="{ 'background-image': 'linear-gradient(0deg, rgba(0,0,0,0) 0%, rgba(46,46,46,0.4) 35%, rgba(0,0,0,0.7) 100%), url(' + value?.article_image + ')' }">
-          <img class="article-logo" :src="value?.article_logo" alt="cows">
-          <h5 class="article-title">{{ value?.article_title }}</h5>
-        </span>
+  <section class="swipe-wrapper" v-if="rowObject">
+    <article
+      class="module-tile has-content"
```

Comment:
i really think using module in this context just confuses things. they are cards, why are they suddenly modules?